Matchismo/Matchismo/Model/SetCardDeck.h

```diff
@@ -0,0 +1,14 @@
+//
+//  SetCardDeck.h
+//  Matchismo
+//
+//  Created by Otto Olkkonen on 15/06/2022.
+//
+
+#import "Deck.h"
+
+@interface SetCardDeck : Deck
+
```

Comment:
We generally try to avoid object-oriented approaches and prefer composition and protocol-oriented approaches wherever possible (just an FYI - I get that this is the approach that the exercise pushes you into).

But in the cases where we do use inheritance, we always try to use the `NS_DESIGNATED_INITIALIZER` macro so that we get as much support from the compiler as possible (you'll also need to combine it with `NS_UNAVAILABLE` in some cases to disallow initialisers from parent classes).